### ISSUE TITLE
Moving windows make them disappear/blink/whatever

### ISSUE THREAD

**Issue by bitstrings** [linuxmint/Cinnamon#8052]
```
 * Cinnamon version 4.0.2
 * Distribution - Manjaro
 * AMD 580 GPU (amdgpu)
 * 64 bit
 * kernel 4.19.2
 ```

Moving windows make them disappear.

Video (4K Screen) -> [HERE](https://drive.google.com/file/d/1G_WCjCnWB7JH8Ofo2Gj6CcB831jYUTEB/view?usp=sharing)

The is NO CLUTTER_* in environment.

In fact this is a fresh install without mod.

I've got a AMD 580 GPU using the free amdgpu.

Also, performance is iffy. Things are a bit jerky. Tried everything.

With or without xorg.conf.

xorg.conf
```
Section "Device"
	Identifier  "Card1"
	Driver      "amdgpu"
	VendorName  "AMD"
	BoardName   "Radeon RX 580"

    Option "TearFree" "on"
    Option "DRI" "3"
    Option "EnablePageFlip" "on"
EndSection
```

**Comment by bitstrings** [comment]
Maybe it's because I have 2 screens one portrait and one landscape.

**Comment by jaszhix** [comment]
I see windows blink occasionally on Nvidia - its always Chromium windows on a second monitor. This has existed pre-4.0. The windows disappearing was a bug that was fixed in Muffin 4.0.2. I think this should be moved to the Muffin repo.
